muted-tests.yml

```diff
@@ -171,4 +171,6 @@ tests:
 #    method: "test {union_types.MultiIndexIpStringStatsInline}"
 #    issue: "https://github.com/elastic/elasticsearch/..."
 #  Note that this mutes for the unit-test-like CsvTests only.
-#  Muting for the integration tests needs to be done for each IT class individually.
+#  Muting all the integration tests can be done using the class "org.elasticsearch.xpack.esql.**".
+#  Consider however, that some tests are named as "test {file.test SYNC}" and "ASYNC" in the integration tests,
+#  so you may need to add the 3 versions, or use a wildcard like "test {file.test*}", which may ignore unexpected tests.
```

Comment:
The wildcard would also cover any test called `file.testOtherCase`.

Another option would be to use 2 mutes: `test {file.test}` and `test {file.test *}`. I'm not sure if it's worth adding it here and removing the current wildcard suggestion (?)

Reply:
Added the example there, using the safe 2 entries way